gf-webapp-admin/src/component/rooms/room/apartmentType/apartmentsList/ApartmentsList.tsx

```diff
@@ -20,8 +19,7 @@ export const ApartmentsList = (props: ApartmentsListProps) => {
   return (
     <select className={styles.list}
       value={props.apartmentsType}
-      disabled={props.isEditFieldDisabled}
-      onChange={(e) => props.onChangeApartmentType(e.target.value as Apartments)}
+      onChange={(e) => props.onChangeValue(e.target.value as Apartments)}
```

Comment:
It's wrong to use as Apartments

Reply:
it's ok in this case

Reply:
Example:  const superPuperConverter (apartmentsRaw: string) => Apartments[apartmentsRaw]

Reply:
It doesnt work